### DISCUSSION TITLE
Header/Footer for the nvim-tree

### DISCUSSION THREAD

**Discussion by Alaz-Oz** [nvim-tree/nvim-tree.lua#1576] [Q&A]
Can we have a header, i.e., heading section for the nvim-tree, where we could put any text like output of the lsp, or count of files open, any custom text. A cutom area? Same with the footer.

**Comment by alex-courtis** [comment]
That is beyond the scope of nvim-tree.

You can use another plugin such as [lualine](https://github.com/nvim-lualine/lualine.nvim) or [airline](https://github.com/vim-airline/vim-airline) to achieve that functionality.
